Exiled.Events/Patches/Generic/PlayerHasHint.cs

```diff
@@ -18,37 +18,28 @@ namespace Exiled.Events.Patches.Generic
 
     using MEC;
 
+    using BaseHint = Hints.Hint;
+
     /// <summary>
-    /// Patches <see cref="HintDisplay.Show(Hint)"/>.
+    /// Patches <see cref="HintDisplay.Show(BaseHint)"/>.
```

Comment:
I know the old patch didn't, but patches summaries should really specify what they are intended to do. IE: Adds x event. Supports x API. Etc.